doc/usage.rst

```diff
@@ -539,6 +539,58 @@ early::
         return new Response(...);
     });
 
+View Handlers
+-------------
+
+View Handlers allow you to intercept a controller result and transform it before
+it gets returned to the kernel.
+
+To register a view handler, pass a callable (or string that can be resolved to a
+callable) to the view method. The callable should accept some sort of result
+from the controller::
+
+    $app->view(function (Response $response) {
```

Comment:
I think this is unfortunate to use a Response here as ann example as this is probably not the common use case.